src/mlpack/core/cv/metrics/r2_score_impl.hpp

```diff
@@ -15,8 +15,9 @@
 namespace mlpack {
 namespace cv {
 
+template<bool AdjustedR2>
 template<typename MLAlgorithm, typename DataType, typename ResponsesType>
-double R2Score::Evaluate(MLAlgorithm& model,
+double R2Score<AdjustedR2>::Evaluate(MLAlgorithm& model,
                          const DataType& data,
                          const ResponsesType& responses)
```

Comment:
```suggestion
double R2Score<AdjustedR2>::Evaluate(MLAlgorithm& model,
                                     const DataType& data,
                                     const ResponsesType& responses)
```
This should make things line up correctly. :+1: